src/helm/helm.ts

```diff
@@ -78,7 +78,7 @@ export async function installHelmChart(
 ): Promise<CliExitData> {
     await syncHelmRepo(repoName);
     return await CliChannel.getInstance().executeTool(
-        HelmCommands.installHelmChart(name, repoName, chartName, version), undefined, false
+        HelmCommands.installHelmChart(name, repoName, chartName, version), undefined, true
```

Comment:
The default value for 2nd and 3rd parameters for `executeTool()` are `undefined` and `true`, so you can remove these two parameters and it will still work.

Reply:
Yes, You're right. Fixed.